dev/com.ibm.ws.microprofile.config.1.1/src/com/ibm/ws/microprofile/config/sources/EnvConfigSource.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017, 2018 IBM Corporation and others.
+ * Copyright (c) 2017, 2020 IBM Corporation and others.
```

Comment:
2018, 2020

Reply:
ignore my last comment. I missed the 2017, bit when I first reviewed.

Reply:
I think this change is correct, 2017 is the original year (which stays the same) while 2020 is the year of the most recent change.